Failure on x86_64-darwin [(full log)](https://logs.nix.ci/?key=nixos/nixpkgs.38408&attempt_id=85ab4ed4-1e0e-4e24-99da-a508a3b6d4f8)

Attempted: monero-gui

<details><summary>Partial log (click to expand)</summary><p>

```
[ 86%] Linking CXX executable ../../bin/monero-blockchain-import
clang-5.0: warning: argument unused during compilation: '-pie' [-Wunused-command-line-argument]
ld: framework not found PCSC
clang-5.0: error: linker command failed with exit code 1 (use -v to see invocation)
make[2]: *** [src/blockchain_utilities/CMakeFiles/blockchain_import.dir/build.make:189: bin/monero-blockchain-import] Error 1
make[1]: *** [CMakeFiles/Makefile2:3126: src/blockchain_utilities/CMakeFiles/blockchain_import.dir/all] Error 2
make: *** [Makefile:141: all] Error 2
builder for '/nix/store/kfh8znhdz79xka2h262128i5dmhwxwqq-monero-0.12.0.0.drv' failed with exit code 2
cannot build derivation '/nix/store/4y1j9nfq8ch7yg2cmwacjl5igxp6wnym-monero-gui-0.12.0.0.drv': 2 dependencies couldn't be built
[31;1merror:[0m build of '/nix/store/4y1j9nfq8ch7yg2cmwacjl5igxp6wnym-monero-gui-0.12.0.0.drv' failed
```
</p></details>

--- a/pkgs/applications/altcoins/monero-gui/default.nix
+++ b/pkgs/applications/altcoins/monero-gui/default.nix
@@ -6,6 +6,7 @@
 , qtwebengine, qtx11extras, qtxmlpatterns
 , monero, unbound, readline, boost, libunwind
 , pcsclite, zeromq, cppzmq, pkgconfig
+, CoreData
 }:
 
 with stdenv.lib;
@@ -30,7 +31,7 @@ stdenv.mkDerivation rec {
     qtxmlpatterns monero unbound readline
     boost libunwind pcsclite zeromq cppzmq
     makeWrapper
-  ];
+  ] ++ stdenv.lib.optional stdenv.isDarwin CoreData;
 
   patches = [
     ./move-log-file.patch

--- a/pkgs/applications/altcoins/monero-gui/default.nix
+++ b/pkgs/applications/altcoins/monero-gui/default.nix
@@ -6,7 +6,6 @@
 , qtwebengine, qtx11extras, qtxmlpatterns
 , monero, unbound, readline, boost, libunwind
 , pcsclite, zeromq, cppzmq, pkgconfig
-, CoreData
 }:
 
 with stdenv.lib;
@@ -31,7 +30,7 @@ stdenv.mkDerivation rec {
     qtxmlpatterns monero unbound readline
     boost libunwind pcsclite zeromq cppzmq
     makeWrapper
-  ] ++ stdenv.lib.optional stdenv.isDarwin CoreData;
+  ];
 
   patches = [
     ./move-log-file.patch

--- a/pkgs/applications/altcoins/monero/default.nix
+++ b/pkgs/applications/altcoins/monero/default.nix
@@ -1,7 +1,8 @@
 { stdenv, fetchFromGitHub, cmake, pkgconfig, git
 , boost, miniupnpc, openssl, unbound, cppzmq
 , zeromq, pcsclite, readline
-, IOKit ? null
+, IOKit
+, CoreData
 }:
 
 assert stdenv.isDarwin -> IOKit != null;
@@ -24,7 +25,7 @@ stdenv.mkDerivation rec {
   buildInputs = [
     boost miniupnpc openssl unbound
     cppzmq zeromq pcsclite readline
-  ] ++ optional stdenv.isDarwin IOKit;
+  ] ++ optionals stdenv.isDarwin [ IOKit CoreData ];
 
   cmakeFlags = [
     "-DCMAKE_BUILD_TYPE=Release"

--- a/pkgs/top-level/all-packages.nix
+++ b/pkgs/top-level/all-packages.nix
@@ -16706,6 +16706,7 @@ with pkgs;
   };
 
   monero-gui = libsForQt5.callPackage ../applications/altcoins/monero-gui {
+    inherit (darwin.apple_sdk.frameworks) CoreData;
     boost = boost15x;
   };
 

--- a/pkgs/top-level/all-packages.nix
+++ b/pkgs/top-level/all-packages.nix
@@ -612,7 +612,7 @@ with pkgs;
   container-linux-config-transpiler = callPackage ../development/tools/container-linux-config-transpiler { };
 
   cconv = callPackage ../tools/text/cconv { };
-  
+
   chkcrontab = callPackage ../tools/admin/chkcrontab { };
 
   djmount = callPackage ../tools/filesystems/djmount { };
@@ -16701,12 +16701,11 @@ with pkgs;
   mod-distortion = callPackage ../applications/audio/mod-distortion { };
 
   monero = callPackage ../applications/altcoins/monero {
-    inherit (darwin.apple_sdk.frameworks) IOKit;
+    inherit (darwin.apple_sdk.frameworks) IOKit CoreData;
     boost = boost15x;
   };
 
   monero-gui = libsForQt5.callPackage ../applications/altcoins/monero-gui {
-    inherit (darwin.apple_sdk.frameworks) CoreData;
     boost = boost15x;
   };
 

